tests/app/main/views/test_suppliers.py

```diff
@@ -300,10 +303,12 @@ def test_should_show_service_details_on_page(self, data_api_client):
     @mock.patch('app.main.views.suppliers.data_api_client')
     def test_should_show_correct_fields_for_disabled_service(self, data_api_client):
         data_api_client.get_supplier.return_value = self.load_example_listing("supplier_response")
+        data_api_client.find_services.return_value = self.load_example_listing("services_response")
+        data_api_client.find_frameworks.return_value = {
+            'frameworks': [self.load_example_listing("framework_response")['frameworks']]
+        }
 
-        services = self.load_example_listing("services_response")
-        services["services"][0]["status"] = "disabled"
-        data_api_client.find_services.return_value = services
+        data_api_client.find_services.return_value["services"][0]["status"] = "disabled"
```

Comment:
Hmm, I'm not super-keen on this (compared to the old version). We've already assigned a return value for the mock above - now we're hacking at a bit of it. This might be personal preference though - I'm willing to be convinced if there's a good reason!

Reply:
Done